app/src/main/java/com/github/se/cyrcle/ui/zone/ZoneManagerScreen.kt

```diff
@@ -23,19 +45,50 @@ import com.github.se.cyrcle.ui.theme.molecules.TopAppBar
 /** Screen where users can manage their downloaded zones. and access the zone selection screen. */
 @Composable
 fun ZoneManagerScreen(navigationActions: NavigationActions) {
+  val context = LocalContext.current
+  val zones = remember { mutableStateOf<List<Zone>>(emptyList()) }
+  LaunchedEffect(Unit) {
+    zones.value = Zone.loadZones(context)
+    Log.d("ZoneManagerScreen", "Zones: ${zones.value}")
+  }
+
   Scaffold(
       topBar = {
         TopAppBar(
             title = stringResource(R.string.zone_manager_screen_title),
             navigationActions = navigationActions)
       }) { padding ->
         Box(modifier = Modifier.padding(padding).fillMaxSize()) {
+          // === Overlay ===
           AddZoneButton(navigationActions)
-          Text("TODO")
+          // ===  === ===  ===
+
+          // === Content ===
+          Column(Modifier.fillMaxWidth()) {
```

Comment:
Here I think a lazycolumn might be a better idea? We do not know the number of zones we have at compile time...

Reply:
Sure I can do a lazyColumn, did not think users would add more than 2 or 3 zones, but it's better